src/gc-pages.c

```diff
@@ -19,6 +19,31 @@ extern "C" {
 #define MIN_BLOCK_PG_ALLOC (1) // 16 KB
 
 static int block_pg_cnt = DEFAULT_BLOCK_PG_ALLOC;
+static _Atomic(size_t) current_pg_count = 0;
```

Comment:
Where is this incremented and decremented?

Reply:
Fixed in the latest commit.